models/lorenz_63/work/input.inflationtest.nml

```diff
@@ -0,0 +1,124 @@
+&filter_nml
+   single_file_in               = .true.
+   input_state_files            = ''
+   input_state_file_list        = 'filter_input_list.txt'
```

Comment:
This whole file is added to help test ... see the instructions in 'inflation_test.m'